tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py

```diff
@@ -994,16 +1014,16 @@ def eltwise_round(
     **kwargs,
 ):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
-    t1 = ttl.tensor.round(t0, decimals, output_mem_config=output_mem_config)
+    t1 = ttl.tensor.rfloor_div(value, t0, output_mem_config=output_mem_config)
```

Comment:
Are we not able to use ttnn here?

Reply:
Hi @eyonland 
rfloor_div was not migrated. Instead we are adding rounding mode floor to drive.